int64[] ar_id
int64[] center_x
int64[] center_y
int64[] theta
